[2024-03-22T23:11:25.652+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag.print_welcome scheduled__2024-03-21T23:00:00+00:00 [queued]>
[2024-03-22T23:11:25.679+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag.print_welcome scheduled__2024-03-21T23:00:00+00:00 [queued]>
[2024-03-22T23:11:25.681+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-22T23:11:25.745+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): print_welcome> on 2024-03-21 23:00:00+00:00
[2024-03-22T23:11:25.750+0000] {standard_task_runner.py:60} INFO - Started process 111 to run task
[2024-03-22T23:11:25.754+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'test_dag', 'print_welcome', 'scheduled__2024-03-21T23:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmprz7j8q0h']
[2024-03-22T23:11:25.756+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask print_welcome
[2024-03-22T23:11:25.873+0000] {task_command.py:423} INFO - Running <TaskInstance: test_dag.print_welcome scheduled__2024-03-21T23:00:00+00:00 [running]> on host 58f158ec64a1
[2024-03-22T23:11:26.060+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_dag' AIRFLOW_CTX_TASK_ID='print_welcome' AIRFLOW_CTX_EXECUTION_DATE='2024-03-21T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-21T23:00:00+00:00'
[2024-03-22T23:11:26.061+0000] {logging_mixin.py:188} INFO - Welcome to Airflow!
[2024-03-22T23:11:26.061+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-22T23:11:26.080+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=print_welcome, execution_date=20240321T230000, start_date=20240322T231125, end_date=20240322T231126
[2024-03-22T23:11:26.130+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-22T23:11:26.195+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
